api/errors/errors.go

```diff
@@ -298,22 +298,38 @@ func NewBlobstoreUnavailableError(cause error) BlobstoreUnavailableError {
 	}
 }
 
-type CFCreateError struct {
+type ResourceNotFoundError struct {
 	apiError
 }
 
-func NewCFCreateError(cause error) CFCreateError {
-	return CFCreateError{
+func NewResourceNotFoundError(cause error) ResourceNotFoundError {
+	return ResourceNotFoundError{
 		apiError: apiError{
 			cause:      cause,
-			title:      "CF-Create",
+			title:      "ResourceNotFound",
 			detail:     cause.Error(),
 			code:       420000,
 			httpStatus: http.StatusInternalServerError,
 		},
 	}
 }
 
+type ResourceNotReadyError struct {
+	apiError
+}
+
+func NewResourceNotReadyError(cause error) ResourceNotReadyError {
+	return ResourceNotReadyError{
+		apiError: apiError{
+			cause:      cause,
+			title:      "ResourceNotReady",
+			detail:     cause.Error(),
+			code:       420001,
```

Comment:
Do we need to amend the other PR with this new code as well?

Reply:
In standup we chatted a bit about this. I don't think we do since it's questionable whether or not `v2.yml` is even meant for that purpose and we've already "reserved" the `420000` block with https://github.com/cloudfoundry/cloud_controller_ng/pull/3205. I think it's worth having that convo there though.

Reply:
Based on the discussion above, I have consolidated the two errors down to just the `ResourceNotReady` error, with a code of `420000` so I think we are good.